### PR TITLE
Resolve endless spamming

### PR DESCRIPTION
**Pull request by wizzymore**
This should fix the issue: https://github.com/azerothcore/mod-buff-command/issues/10

This needs to be tested!

### CODE REVIEW

File: src/BuffCommand.cpp [hunk 1]
```diff
@@ -83,8 +104,25 @@ class Kargatum_BuffLoad : public WorldScript
 	}
 };
 
+class Kargatum_Config : public WorldScript
```
Comment (Winfidonarleyan) [inline src/BuffCommand.cpp]: No need this script, command in CMake `AC_ADD_CONFIG_FILE` added config before

File: src/cmake/after_ws_install.cmake [hunk 2]
```diff
@@ -0,0 +1,15 @@
+if( WIN32 )
```
Comment (Winfidonarleyan) [inline src/cmake/after_ws_install.cmake]: No need this file

**Comment by wizzymore** [comment]
Thank you @Winfidonarleyan

**Comment by wizzymore** [comment]
Currently you can double .buff so i have to fix this

**Comment by Winfidonarleyan** [comment]
Delete full file `after_ws_install.cmake`

**Comment by wizzymore** [comment]
On linux without that file you won't have the .conf.dist in the ETC folder

In windows should have the same feature

Why should i remove it?

**Comment by Winfidonarleyan** [comment]
https://github.com/azerothcore/azerothcore-wotlk/blob/b3876461e8e4333c5887d761288da4beea223b06/src/server/worldserver/CMakeLists.txt#L110

**Comment by Winfidonarleyan** [comment]
Learn commit https://github.com/azerothcore/azerothcore-wotlk/commit/d29a1a330018b89ee608aa27e500068f5310ed2a

**Comment by wizzymore** [comment]
Ok, i got it now, i will remove the file

File: CMakeLists.txt [hunk 3]
```diff
@@ -1,3 +1,6 @@
+CU_SET_PATH("CMAKE_MOD_BUFFCOMMAND_DIR" "${CMAKE_CURRENT_LIST_DIR}")
```
Comment (wizzymore) [inline CMakeLists.txt]:
```diff
-CU_SET_PATH("CMAKE_MOD_BUFFCOMMAND_DIR" "${CMAKE_CURRENT_LIST_DIR}")
+CU_SET_PATH("MODULE_CONFIG_FILE_LIST" "${CMAKE_CURRENT_LIST_DIR}")
```

This is needed, right?

**Review by wizzymore — COMMENTED** [review]
@Winfidonarleyan

File: CMakeLists.txt (hunk 3 above)
Comment (wizzymore) [reply CMakeLists.txt]: Okay, i got it now, from that commit

Comment (wizzymore) [reply CMakeLists.txt]: https://github.com/azerothcore/azerothcore-wotlk/blob/d29a1a330018b89ee608aa27e500068f5310ed2a/src/cmake/ac_macros.cmake#L26

**Review by Winfidonarleyan — APPROVED** [review]

**Pull request merged**